website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends controller-runtime's [envtest][envtest] because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
```

Comment:
I think including a blurb here to stress that this is about building tests for your generated operator, not for operatorSDK itself. Something like "The Operator SDK project recommends using controller-runtime's [envtest][envtest] to write tests for your operatorbecause it has a more active contributor community..."

Reply:
I updated it here and in the migration guide since it is how we added the info there as well.

Reply:
should be "for your Operator project" but other than that looks good